prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh

```diff
@@ -102,6 +117,12 @@ cleanup() {
         az group delete -n "${RS_GROUP}" -y
     fi
 
+    if [ -n "${CLEANUP_DOCKER_IMAGE}" ]; then
+        shout "Delete temporary Kyma-Installer Docker image"
+        date
+        "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-image.sh"
+    fi
+
     rm -rf "${TMP_DIR}"
```

Comment:
I see `TMP_DIR=$(mktemp -d)` was added, however, `rm -rf "${TMP_DIR}"` was already there. So I am wondering if TMP_DIR was already set before 🤔 If yes, then we are not deleting the previous TMP_DIR anymore.

Reply:
I could not find any trace of a previous tmp_dir, lets see what happens once we execute the script... local testing is a bit limited for those scripts

Reply:
on the other hand... this is meant to be executed in a docker container so the cleanup of folders is kinda useless, don't you think?